traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -1645,6 +1645,15 @@ def get_server_delivery_services(self, server_id=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	@restapi.api_request('get', 'servers/details?hostName={name}', ('3.0','4.0',))
+	def get_server_details(self, name=None):
+		"""
+		Get servers/details
+		:ref: `to-api-servers-details-hostName`
```

Comment:
This label, `to-api-servers-details-hostName`, doesn't exist in the documentation. Closest is `to-api-servers-details`, which I think is what you want.

Also, there shouldn't be a space between the `ref` role and its argument in grave accents ("backticks").

Reply:
My mistake. I mentioned in reference to GET request: https://traffic-control-cdn.readthedocs.io/en/latest/api/v4/servers_details.html#to-api-servers-details
GET /api/4.0/servers/details?hostName but yes, endpoint is servers/details. Updated now.